flink-staging/flink-table/pom.xml

```diff
@@ -37,7 +37,7 @@ under the License.
 		<dependency>
 			<groupId>com.google.guava</groupId>
 			<artifactId>guava</artifactId>
-			<version>${guava.version}</version>
+			<version>14.0.1</version>
```

Comment:
Yes, I had method signature problems with the HCat input format. We are using a very old version of HCat and therefore need a very old version of guava.